packages/@expo/fingerprint/cli/src/runLegacyCLIAsync.ts

```diff
@@ -0,0 +1,47 @@
+import readFingerprintFileAsync from './utils/readFingerprintFileAsync.js';
+import {
+  createFingerprintAsync,
+  diffFingerprintChangesAsync,
+  diffFingerprints,
+} from '../../build/index.js';
+
+export async function runLegacyCLIAsync(args: string[]) {
+  if (args.length !== 1 && args.length !== 2 && args.length !== 3) {
```

Comment:
nit: if the legacy cli has some maintaining cost, we can just say the legacy cli has been deprecated.

Reply:
Eh, it's not too bad. Only risk would be someone running the legacy CLI on a directory called `fingeprint:generate`, but that's exceedingly unlikely.